docs/how-to/system-optimization/mi300a.rst

```diff
@@ -145,15 +145,26 @@ This section describes performance-based settings.
      sudo modprobe amdttm pages_limit=134217728 page_pool_size=134217728
      sudo modprobe amdgpu
 
-  These settings can also be hardcoded in the ``/etc/modprobe.d/amdttm.conf`` file. To use this method, 
+  These settings can also be hardcoded in the ``/etc/modprobe.d/amdttm.conf`` file or specified as boot
+  parameters.
+  
+  To use the hardcoded method, 
   the filesystem must already be set up when the kernel driver is loaded.
-  Add the following lines to ``/etc/modprobe.d/amdttm.conf``:
+  To hardcode the settings, add the following lines to ``/etc/modprobe.d/amdttm.conf``:
 
   .. code-block:: shell
 
      options amdttm pages_limit=134217728
      options amdttm page_pool_size=134217728
 
+  If the filesystem is not already set up when the kernel driver is loaded, then the options
+  must be specified as boot parameters. To specify the settings
+  as boot parameters when loading the kernel, use this example as a guideline:
+
+  .. code-block:: shell
+
+     vmlinux-[...] amdttm.pages_limit=134217728 amdttm.page_pool_size=134217728 [...]
+
```

Comment:
Thanks, this looks good to me.  We could get more eyes on it, but this would only introduce further delay and they missed these details earlier regardless.  We'll look for feedback from users of the documentation to guide us if further improvements can be made.